### DISCUSSION TITLE
Need a custom disconnect hook which i can call directly on logout(my app logout).

### DISCUSSION THREAD

**Discussion by RahulAtRecorem** [rainbow-me/rainbowkit#620] [Ideas]
I don't want the user to open accountModel and then click disconnect manually, I want to call disconnect on logout. can we add that disconnect hook, please?

I need something like a disconnectWallet hook which I can call while logout flow.

If something like this is already there, please do let me know. it will help a lot

**Comment by markdalgleish** [comment]
This can be done with wagmi's `useDisconnect` Hook: https://wagmi.sh/docs/hooks/useDisconnect
